### PR TITLE
exploding va_rx gem into vets-api -- with minor refactor

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

File: spec/lib/common/models/base_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require_dependency 'common/models/base'
+
+# FIXME: Add this spec
+describe Common::Base do
+end
```
Comment (lihanli) [inline spec/lib/common/models/base_spec.rb]: did you forget to do this or are you going to add it in another pr?

**Comment by kreek** [comment]
Other than @lihanli 's comment LGTM

File: spec/lib/common/models/base_spec.rb (hunk 1 above)
Comment (saneshark) [inline spec/lib/common/models/base_spec.rb]: it will be in a separate PR. Currently have close to 100% coverage, but it's based off of Prescriptions. I think it should be more generic than that, but didn't want to spend umpteen hours rewriting specs when it's already at 100%

**Comment by lihanli** [comment]
:+1:

File: app/models/prescription.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  attribute :is_refillable, Boolean
+  attribute :is_trackable, Boolean
+
+  alias refillable? is_refillable
+  alias trackable? is_trackable
+
+  def <=>(other)
+    return -1 if prescription_id < other.prescription_id
+    return 1 if prescription_id > other.prescription_id
+    0
```
Comment (carlmjohnson) [inline app/models/prescription.rb]: Can this be shortened to `prescription_id <=> other.prescription_id`?

File: lib/common/client/configuration.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Common
+  module Client
+    # Configuration class used to setup the environment used by client
+    class Configuration
+      OPEN_TIMEOUT = 15
```
Comment (carlmjohnson) [inline lib/common/client/configuration.rb]: Add a comment with units. Seconds?

File: app/models/prescription.rb (hunk 2 above)
Comment (saneshark) [inline app/models/prescription.rb]: maybe... (it should) let me test that out really quick -- will make similar change for tracking if its the case.

File: lib/common/client/configuration.rb (hunk 3 above)
Comment (saneshark) [inline lib/common/client/configuration.rb]: sure, i can do that -- ping me when you're done reviewing.
